auth/src/ios/auth_ios.mm

```diff
@@ -590,6 +590,19 @@ void SignInCallback(FIRUser *_Nullable user, NSError *_Nullable error,
   SetEmulatorJni(auth_data_, host.c_str(), port);
 }
 
+AuthError Auth::UseUserAccessGroup(const char* access_group) {
+  if (!auth_data_) {
+    return kAuthErrorUninitialized;
```

Comment:
This error code doesn't exist, please check the AuthError enum and choose a more suitable one.